### DISCUSSION TITLE
ESP32 Tasmota - Software serial ?

### DISCUSSION THREAD

**Discussion by dexterbot80** [arendst/Tasmota#20844] [Q&A]
Hello, I want to connect a sensor to the RX TX pins of an ESP32. I installed Tasmota and selected pins 26 and 27 from the list, but it doesn't seem to work. Do the pins have to be initialized somehow to work on 115200 baud rate?
Thank you
<img width="482" alt="image" src="https://github.com/arendst/Tasmota/assets/47325794/92c75601-63ed-4618-937e-1f627e296f9c">

**Comment by sfromis** [comment]
The other thread https://github.com/arendst/Tasmota/discussions/18340 looks like you found the solution, to use SerBr TX/RX instead of Serial TX/RX, and do a send first. Default serial setup should be 115200 8N1, until you change it.

**Reply by dexterbot80** [reply]
I receive the correct messages using esp8266 but i want to use the fastest esp32
